README.rst

```diff
@@ -42,21 +42,17 @@ Installation
 ============
 
 We recommend the `Anaconda Python distribution <https://www.anaconda.com/products/individual>`_.
-To install ``hnn-core``, you first need to install its dependencies::
+To install ``hnn-core``, simply do::
 
-	$ conda install numpy matplotlib scipy
+   $ pip install hnn_core
 
-Additionally, you would need Neuron which is available here:
-`https://neuron.yale.edu/neuron/ <https://neuron.yale.edu/neuron/>`_.
-Note installing Neuron using the traditional installer may make it hard to manage
-within virtual environments such as Anaconda. For this reason, we recommend the
-pip installer:
-
-	$ pip install NEURON
+and it will install ``hnn-core`` along with the dependencies which are not already installed.
 
-Since ``hnn-core`` does not yet have a stable release, we recommend installing the nightly version. This may change in the future if more users start using it.
+Note that if you installed Neuron using the traditional installer package, it is recommended
+to remove it first and unset ``PYTHONPATH`` and ``PYTHONHOME`` if they were set. This is
+because the pip installer works better with virtual environments such as Anaconda.
```

Comment:
```suggestion
because the pip installer works better with virtual environments such as provided by `conda`.
```

Reply:
not sure about my suggestion, your call... In my head, Anaconda is a meta-package managed by conda.